sources/packages/forms/src/form-definitions/studentfinancialinformationappeal.json

```diff
@@ -89,6 +89,1174 @@
+                    "refreshOnChange": false,
+                    "customClass": "",
+                    "hidden": false,
+                    "modalEdit": false,
+                    "key": "html87",
+                    "tags": [],
+                    "properties": {},
+                    "conditional": {
+                        "show": "",
+                        "when": null,
+                        "eq": "",
+                        "json": ""
+                    },
+                    "customConditional": "",
+                    "logic": [],
+                    "attributes": {},
+                    "overlay": {
+                        "style": "",
+                        "page": "",
+                        "left": "",
+                        "top": "",
+                        "width": "",
+                        "height": ""
+                    },
+                    "type": "htmlelement",
+                    "input": false,
+                    "placeholder": "",
+                    "prefix": "",
+                    "suffix": "",
+                    "multiple": false,
+                    "defaultValue": null,
+                    "protected": false,
+                    "unique": false,
+                    "persistent": false,
+                    "clearOnHide": true,
+                    "refreshOn": "",
+                    "redrawOn": "",
+                    "tableView": false,
+                    "dataGridLabel": false,
+                    "labelPosition": "top",
+                    "description": "",
+                    "errorLabel": "",
+                    "tooltip": "",
+                    "hideLabel": false,
+                    "tabindex": "",
+                    "disabled": false,
+                    "autofocus": false,
+                    "dbIndex": false,
+                    "customDefaultValue": "",
+                    "calculateValue": "",
+                    "calculateServer": false,
+                    "widget": null,
+                    "validateOn": "change",
+                    "validate": {
+                        "required": false,
+                        "custom": "",
+                        "customPrivate": false,
+                        "strictDateValidation": false,
+                        "multiple": false,
+                        "unique": false
+                    },
+                    "allowCalculateOverride": false,
+                    "encrypted": false,
+                    "showCharCount": false,
+                    "showWordCount": false,
+                    "allowMultipleMasks": false,
+                    "id": "eui7cei",
+                    "addons": []
+                },
+                {
+                    "label": "<strong>Do you want to be assessed with this year's estimated income?</strong>",
```

Comment:
Please remove the `<strong>` from labels. I am currently working to remove them all from every other form. They are displayed in the validation errors and they are not needed. Removing them should not affect the UI.